extensions/vscode/package.json

```diff
@@ -160,7 +160,7 @@
     "@types/mocha": "^10.0.3",
     "@types/node": "20.8.6",
     "@types/vscode": "^1.79.0",
-    "@typescript-eslint/eslint-plugin": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^6.9.0",
```

Comment:
`import * as Mocha from "mocha";` is throwing a warning on `index.ts`.

We should amend the lint change made in this Pull Request to not throw there. The code is from the VS Code template sample and I prefer having it consistency with it for future reference. Although, I'm not too fussed if we rename it to follow camelCase.